frameworks/person-escort-record/README.md

```diff
@@ -106,6 +121,9 @@ Supported validations:
 
 - `required` - mark a question as mandatory. For radios or checkboxes it will require at least one option to be selected, for text or textarea questions it will require at least 1 character
 
+- `min_length` - mark the minimum length of items required for a question with a response of type list. This validation requires an extra `value` option which specifies the length as an integer. If not specified defaults to `0`.
```

Comment:
Could we remove the `_` from these? It's a little FE selfish but it means I don't need to do extra transformation when passing through to the form wizard and using [the validations](https://github.com/UKHomeOffice/passports-form-wizard/blob/9dd5356e7d62c3c4050ed83e43a8a2c543577429/lib/validation/validators.js#L29) from there that are already defined.

Reply:
Maybe we should also mention in this validations section that it is only support on particular question types.

Reply:
Amended, re the particular question types, I've added `question with a response of type list` to allow this to be used for any questions which could have a list of responses instead of specifying `add_multiple`. In the question type question I do mention those validations